packages/editor/src/systems/ClickPlacementSystem.tsx

```diff
@@ -143,9 +143,9 @@ const ClickPlacementReactor = (props: { parentEntity: Entity }) => {
         (uuid) => uuid !== getComponent(clickState.placementEntity.value, UUIDComponent)
       )
       EditorControlFunctions.removeObject([clickState.placementEntity.value])
-      removeEntity(clickState.placementEntity.value)
       clickState.placementEntity.set(UndefinedEntity)
       SelectionState.updateSelection(selectedEntities)
+      applyIncomingActions()
```

Comment:
Running applyIncomingActions here will likely have unintended consequences, what is the reason for doing this?

Reply:
Ok, I'll try to dig more into the root cause, at the base it seems like there's a race condition with removeObject where there's a frame in the GLTFSnapshotState where the entity no longer exists, but is still listed in the GLTF nodes